modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractKotlinCodegen.java

```diff
@@ -71,6 +71,7 @@
     public AbstractKotlinCodegen() {
         super();
         supportsInheritance = true;
+        setSortModelPropertiesByRequiredFlag(true);
```

Comment:
I suggest not to set it as the default:

1) it will be a breaking change and can't target master.
2) not everyone wants this to be the default behavior

Reply:
As discussed, it's not a breaking change as you've updated the template as well